### ISSUE TITLE
ok zoomer  10.0.0-beta.5+fabric can't not work on mincraft 1.21

### ISSUE THREAD

**Issue by StarsShine11904** [joaoh1/OkZoomer#123]
fabric api 0.106.0 was not launched in 1.21, but the module requires this version

Can you fix this problem?

```
A potential solution has been determined, this may resolve your problem:
	 - Replace mod 'Ok Zoomer' (ok_zoomer) 10.0.0-beta.5 with any version that is compatible with:
		 - fabric-api 0.102.0+1.21
More details:
	 - Mod 'Ok Zoomer' (ok_zoomer) 10.0.0-beta.5 requires version 0.106.0 or later of mod 'Fabric API' (fabric-api), but only the wrong version is present: 0.102.0+1.21!
	 - Mod 'Tinfoiled Cotton Client Commands' (cotton-client-commands) 1.1.0+1.15.2 requires any version between 1.14- (inclusive) and 1.16- (exclusive) of 'Minecraft' (minecraft), but only the wrong version is present: 1.21!
```

**Comment by UpcraftLP** [comment]
Thanks for notifying, I marked it as 1.21.1+ only for now.

Will push a fixed version in a bit.

**Comment by EnnuiL** [comment]
Oh whoops! Another hand here, I've accidentally remarked said version to 1.21-compatible :p (this has been rectified)
I'll let Up cook here; also I'll investigate other versions of Ok Zoomer to see if it the 1.21-compatible label is also invalid there

Update: yeah, it was correct for 10.0.0-beta.4 because it wasn't multiloader yet; I forgot to downgrade the minimum requirements for 1.21 when backporting it; Up's already handling this

**Comment by UpcraftLP** [comment]
Fixed in 10.0.0-beta.6
